CHANGELOG.md

```diff
@@ -3,6 +3,117 @@ All notable changes to this project will be documented in this file.
+- alpaka's memory fences can now also be applied to the grid level #1641
+- `alpaka::getWarpSize()` was renamed to `alpaka::getWarpSizes()` and will now return a `std::vector` of supported warp sizes #1644
+- Documentation updates:
+  - Improved installation and usage documentation #1571
+  - Added documentation on how to write unit tests #1609
+  - The HIP portion of the compiler support matrix has been simplified #1637
+
+### Misc:
+- The CUDA and HIP back-ends no longer explicitly set the device where this is unnecessary #1515
+- `clang-tidy`'s modernization suggestions have been applied to the code base #1584
+- alpaka's math headers have been squashed. For each back-end there is now only one header instead of one for each math function. #1585
+- Updated the Boost predefinition header to reflect the upgrade to Boost 1.74 #1586
+- Removed the `alpaka::extent` namespace (the contents now live in the main `alpaka` namespace) #1593
+- Refactored implementations of `BufCpu` and `BufUniformCudaHipRt` #1608
+- Removed unnecessary specializations of `GetPitchBytes` trait #1614
+- All alpaka-specific CMake variables follow the `${PROJECT_NAME}_VARIABLE_FOO_BAR` pattern. This means that all alpaka-specific CMake variables look like this: `alpaka_VARIABLE_FOO_BAR`. #1653
+- alpaka now enforces that kernel arguments are trivially copyable #1635
+- Renamed namespace `traits` to `trait` #1651
+- alpaka now enforces that kernel functions are trivially copyable #1654
+
+### Breaking Changes
+- C++14 is no longer supported (see above)
+- alpaka now uses Boost.Atomic by default if the latter can be found by CMake. This can be turned off by passing `-DALPAKA_ACC_CPU_DISABLE_ATOMIC_REF=ON` during the CMake configuration phase. #1566
+- Removed the `alpaka::extent` namespace (the contents now live in the main `alpaka` namespace) #1593
+- Kernel arguments are required to be trivially copyable. This was always a requirement but is now enforced by alpaka #1635
+- All alpaka-specific CMake variables follow the `${PROJECT_NAME}_VARIABLE_FOO_BAR` pattern. This means that all alpaka-specific CMake variables look like this: `alpaka_VARIABLE_FOO_BAR`. #1653
+- `alpaka::getWarpSize()` was renamed to `alpaka::getWarpSizes()` and now returns a vector of supported warp sizes #1644
+- `alpaka::clock()` is now deprecated and will be removed in the next release. The compiler will warn about its usage. #1645
+- Renamed namespace `traits` to `trait` #1651
+- Removed support for `std::function` kernel functions #1654
+- Kernel functions are required to be trivially copyable. With the exception of `std::function` (see bullet point directly above) this was always a requirement but is now enforced by alpaka. #1654
+
+### Test Cases / CI:
+- Added clang 11, 12 + CUDA 11 tests to CI #1466
```

Comment:
```suggestion
- Added clang 11/12 + CUDA 11 tests to CI #1466
```

Reply:
I noted these down, will change in the next iteration of `CHANGELOG.md` (since a few minor things will probably still make it into the release).

Reply:
ok!